lib/v2/xmut/jwc/bkjw.js

```diff
@@ -0,0 +1,64 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const xmut = 'https://jwc.xmut.edu.cn';
+
+module.exports = async (ctx) => {
+    const { category = 'jwxt' } = ctx.params;
+    const url = `${xmut}/index/tzgg/${category}.htm`;
+    const res = await got(url, {
+        headers: {
+            referer: xmut,
+        },
+        https: {
+            rejectUnauthorized: false,
+        },
+    });
+
+    const $ = cheerio.load(res.data);
+    const itemsArray = [];
+
+    $('#result_list table tbody tr').each((index, row) => {
+        const res = $('td', row).eq(0);
+        const resDate = $('td', row).eq(1);
+        const resLink = $('a', res).attr('href');
+        let link;
+
+        if (resLink.startsWith('../../')) {
+            link = `${xmut}/${resLink}`;
+        } else {
+            link = resLink;
+        }
+
+        const title = $('a', res).attr('title');
+        const pubDate = parseDate(resDate.text().trim());
+
+        itemsArray.push({
+            title,
+            link,
+            pubDate
+        });
+    });
```

Comment:
Please use [map()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create arrays instead of a loop and push.